postgres/datadog_checks/postgres/postgres.py

```diff
@@ -361,6 +386,7 @@ def _load_query_max_text_size(self, db):
                 "dd.postgres.error",
                 1,
                 tags=self._config.tags + ["error:load-track-activity-query-size"],
+                hostname=self.resolved_hostname,
```

Comment:
We should add the hostname to the other internal DBM `dd.postgres.*` metrics in `statements.py` and `statement_samples.py` as well.

Reply:
Do we definitely want that? Should I send the `agent_hostname` tag as well so we can still debug the specific agent?

Reply:
> Do we definitely want that?

It'll make it easier to troubleshoot as then these debug metrics will have the same set of tags as the DBM telemetry. 

> Should I send the `agent_hostname` tag as well so we can still debug the specific agent?

Yes good idea. Could add as a separate tag.